fix(docs): Fixed Typo in language lead handbook

Nishant-96 opened the pull request:
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.


Closes #46977

The issue was to fix two typos first one at line 34 where `there two places` should have been `there are two places` and second one at line 269 where the word `settngs` was misspelled. 

This PR fixes both the typos.

![image](https://user-images.githubusercontent.com/87017414/180305526-d75341cb-39a3-4e62-ac02-9cf76877ca04.png)

![image](https://user-images.githubusercontent.com/87017414/180305049-e3ceffa2-7ecb-4d9e-adda-4d04d29ef5cc.png)

Nishant-96 committed:
fix(docs): Fixed Typo in language lead handbook

Sboonny reviewed (dismissed):
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @Nishant-96, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

ojeytonwilliams commented on docs/language-lead-handbook.md:
```diff
@@ -266,7 +266,7 @@ Then there are three steps to complete:
 ![pre-translate existing translations](./images/crowdin/pre-translate3.png)
 
-When you have finished settngs this, press the Pre-Translate button, and wait. It will alert you once it has finished. It will take more or less time depending on how many untranslated strings are present in the files you have choosen.
+When you have finished settings this, press the Pre-Translate button, and wait. It will alert you once it has finished. It will take more or less time depending on how many untranslated strings are present in the files you have choosen.
```
```suggestion
When you have finished setting this, press the Pre-Translate button and wait. It will alert you once it has finished. The time it takes depends on how many untranslated strings are in the chosen files.
```
Since we're fixing things, this sentence is a little wordier than it needs to be.

Nishant-96 committed:
docs: Updated docs language-lead-handbook

updated line 269 making it more concise as suggested.

Co-authored-by: Oliver Eyton-Williams <ojeytonwilliams@gmail.com>

Sboonny commented:
Second I want to add a co-author

Commit:
Adding cooo-author

Co-authored-by: Hrittik Garain <84389562+hrittik2002@users.noreply.github.com>

Commit:
revert the file back

Nishant-96 commented:
Hey @Sboonny  Just wanted to confirm if i missed something since there has been no update on this pR. I fixed both the typos and implemented the requested suggestion.

Sboonny commented:
Nah @Nishant-96, it's weekend and people take their break.

ojeytonwilliams commented:
LGTM, thanks @Nishant-96 :+1:
